REFERENCE.md

```diff
@@ -524,16 +538,38 @@ Data type: `Optional[Array]`
 
 Default value: ``undef``
 
+##### <a name="ensure"></a>`ensure`
+
+Data type: `Enum['present', 'absent']`
+
+
+
+Default value: `'present'`
+
 ### <a name="telegrafinput"></a>`telegraf::input`
 
-The telegraf::input class.
+== Define: telegraf::input
+
+A Puppet wrapper for discrete Telegraf input files
+
+=== Parameters
+
+[*options*]
```

Comment:
Looks like this uses an old style of parameter documentation. Might be nice to convert to puppet-strings.